recipes/urdfdom_headers/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+mkdir build
+cd build
+
+cmake .. \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DCMAKE_PREFIX_PATH=$PREFIX -DCMAKE_INSTALL_PREFIX=$PREFIX \
+      -DCMAKE_INSTALL_LIBDIR=lib
```

Comment:
Should it be ``-DCMAKE_INSTALL_LIBDIR=$PREFIX/lib`` ?

Reply:
I am not a cmake expert by any means, but I have seen [this](https://github.com/conda-forge/arrow-cpp-feedstock/issues/61) for example that seem to indicate that `lib` is preferred.

Reply:
No, `lib` is correct, `CMAKE_INSTALL_PREFIX` will be prepended by CMake.

Reply:
Great to have the confirmation, thanks!